src/app/components/InlineLink/README.md

```diff
@@ -0,0 +1,20 @@
+# Inline Link [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=InlineLink)
+
+## Description
+
+The `Inline Link` component is made up of a styled HTML `<a>` element with an `href` attribute. Links open in the current tab. 
+
+## When to use this component
+
+The purpose of this component is to return an inline link element.
+
+`Inline Link` is made use of indirectly in the [Text](../Text) component, which uses the [Inline Link Container](../../containers/InlineLink) to render an `Inline Link` differently based on whether the href points in or outside of Simorgh.
```

Comment:
>differently based on whether the href points in or outside of Simorgh.

I don't think we need to mention this because this read me is about the generic component not our bespoke implementation of it. This kind of context should be in the `Inline Link Container readme` which handles the logic for internal/external Simorgh URLs.